mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/ConcurrentExistsResponse.test

```diff
@@ -102,7 +102,7 @@ SESSION: 4
 C: 4b CREATE another
 S: \* 1 EXISTS
 S: \* 1 RECENT
-S: \* 1 FETCH \(FLAGS \(\\Flagged \\Recent\)\)
+S: \* 1 FETCH \(FLAGS \(\\Recent\)\)
```

Comment:
Session 4 selected the existsresponse mailbox on line 51

Then Session one append a message

By this create, we generate an unsolicited response that is added before the command output. Akka like Noop. What it prooves here is that we lost track of ApplicableFlags changes on selected mailbox.